Resources/public/css/views/serverside.css

```diff
@@ -2,7 +2,97 @@
  * Copyright (C) eZ Systems AS. All rights reserved.
  * For full copyright and license information view LICENSE file distributed with this source code.
  */
-
 .ez-view-serversideview .ez-platform-version {
     margin-top: 0;
 }
+
+.ez-view-serversideview .ez-serverside-content {
+    padding-top: 0px;
+}
+
+.ez-view-serversideview .ez-serverside-content-inner {
+    padding: 0.5em 1em;
+}
+
+.ez-view-serversideview .ez-serverside-content-inner > section > section {
```

Comment:
a tad too specific (in addition to the inner class name already mentioned below), so you should add a class on the section you want to style (ez-field-definition-group ?), so this would give:
```css
.ez-view-serversideview .ez-field-definition-group {
    /* ... */
}
```
.ez-serverside-content-inner is not needed if the class name is specific enough (hence why it's important to respect the domain)